lib/js/components/Icon/Icon.stories.ts

```diff
@@ -0,0 +1,53 @@
+import Icon from './Icon.vue';
+import { ICON_SIZES, ICONS } from './Icon.consts';
+
+import { Args, ArgTypes, Meta, StoryFn } from '@storybook/vue';
+
+export default {
+	title: 'Components/Icon',
+	component: Icon,
+} as Meta<typeof Icon>;
+
+const StoryTemplate: StoryFn<typeof Icon> = (argTypes) => ({
+	components: { Icon },
+	props: Object.keys(argTypes),
+	template:
+		'<div class="iconList__singleIcon"><wnl-icon :icon="ICONS[icon]" :size="size" :touchable="touchable" :spinning="spinning" :rotation="rotation" :flipped-vertical="flippedVertical" :flipped-horizontal="flippedHorizontal" /></div>',
+});
+
+export const Interactive = StoryTemplate.bind({});
+
+const args = {
+	size: ICON_SIZES.MEDIUM,
+	icon: 'HEAD_WITH_QUESTION_MARK',
+	spinning: false,
+	touchable: false,
+	rotation: null,
+	flippedVertical: false,
+	flippedHorizontal: false,
+} as Args;
+
+const argTypes = {
+	size: {
+		control: { type: 'select', options: Object.values(ICON_SIZES) },
+		defaultValue: ICON_SIZES.MEDIUM,
+	},
+	icon: {
+		control: { type: 'select', options: Object.keys(ICONS) },
+		defaultValue: 'HEAD_WITH_QUESTION_MARK',
```

Comment:
this story have some error and cannopt be rendered on my dev env

Reply:
Fixed.